### ISSUE TITLE
Buzz! Master Quiz has problem with video clip

### ISSUE THREAD

**Issue by Lince90** [hrydgard/ppsspp#6190]
Hi!
I have some problem with this game....when there is a question with a video i can't see it but i listen the sound...please fix it!

**Comment by daniel229** [comment]
The video seems play,but missing sound.
![01](https://cloud.githubusercontent.com/assets/3481559/5987288/4e8d0b68-a95d-11e4-9086-93dd834f7354.png)

```
23:13:020 user_main    W[ME]: HLE\scePsmf.cpp:1793 scePsmfPlayerChangePlayMode(08defc4c, 0, 0): play speed not changed
23:13:038 user_main    W[ME]: HLE\scePsmf.cpp:1793 scePsmfPlayerChangePlayMode(08defc4c, 0, 0): play speed not changed
23:13:053 user_main    W[ME]: HLE\scePsmf.cpp:1793 scePsmfPlayerChangePlayMode(08defc4c, 0, 0): play speed not changed
23:13:070 MovieAudioTh I[ME]: HW\MediaEngine.cpp:85 FF: Frame data doesn't match channel configuration!
23:13:070 MovieAudioTh E[ME]: HW\SimpleAudioDec.cpp:201 Error decoding Audio frame (376 bytes): -1094995529 (bebbb1b7)
23:13:070 MovieAudioTh E[ME]: HW\MediaEngine.cpp:803 Audio (AT3+) decode failed during video playback
```

**Comment by unknownbrackets** [comment]
Is this also affected by that mono thing like the other game (#3844)?

-[Unknown]

**Comment by daniel229** [comment]
Yes,it's mono,but it seems that hack break other thing,still no sound.
after apply that hack
got this
```
user_main    W[ME]: HLE\scePsmf.cpp:1793 scePsmfPlayerChangePlayMode(08defc4c, 0, 0): play speed not changed
```

**Comment by hrydgard** [comment]
#8472 probably helped?

**Comment by daniel229** [comment]
Yes,fixed

**Comment by daniel229** [comment]
http://forums.ppsspp.org/showthread.php?tid=17873 ,Look like the video still not showing up.

![01](https://cloud.githubusercontent.com/assets/3481559/13039407/818c7a38-d3d9-11e5-97ed-d25e534f4756.png)

**Comment by daniel229** [comment]
It catches a frame and flickering coming back from pause menu.
![02](https://cloud.githubusercontent.com/assets/3481559/13039530/125bfae2-d3db-11e5-9bb3-575c6618cb62.png)

**Comment by daniel229** [comment]
The videos show up after https://github.com/jpcsp/jpcsp/commit/2122e18ccf22f088860be8307a78b40ce2d0e624 in jpcsp.

**Comment by benderscruffy** [comment]
this can be closed the videos play with sound

**Comment by megas7** [comment]
I have the same problem i can listen the video but i can't see it

**Comment by megas7** [comment]
![Screenshot_429](https://user-images.githubusercontent.com/28945155/77867447-e9d86300-723f-11ea-8bb4-002627048fca.png)

**Comment by unknownbrackets** [comment]
Perhaps this is an issue with certain regions of the game specifically?

Are you using default settings?  Simulate block transfer is probably required, and "skip buffer effects" almost definitely will break the video.

-[Unknown]

**Comment by megas7** [comment]
i just did it as you was metntion and i got black sceen when was playing the video and the audio only played 
![Screenshot_430](https://user-images.githubusercontent.com/28945155/77867961-22793c00-7242-11ea-9690-9fef264b4b13.png)

**Comment by megas7** [comment]
the same on switch 
![2020033005033600-0E7DF678130F4F0FA2C88AE72B47AFDF](https://user-images.githubusercontent.com/28945155/77868511-23ab6880-7244-11ea-837e-324f8f188956.jpg)

**Comment by megas7** [comment]
also on windows i click it fast to go to fullsceen and to go back on window and i got some image from that but the video is not working for 1 sec only

**Comment by Gamemulatorer** [comment]
Video work only in software.

https://user-images.githubusercontent.com/37603562/163707776-c4054a61-1338-4c04-bd3b-2afd6edb7b58.mp4

Still no video on vulkan but has a sounds.

https://user-images.githubusercontent.com/37603562/163707795-b057d5f0-ce7d-45bb-8802-54b635289c3e.mp4

**Comment by Feregorn** [comment]
Hello, same issue here. With Vulkan on Android there is no video playing during those question, only sound can be heard. When selecting software then the videos play normally but the performance drops significantly and the gameplay becomes choppy. Is there a way to apply what software rendering fixes during those questions to the Vulkan rendered?

**Comment by sum2012** [comment]
v1.15.4-60-ge1890b6a9 no video on vulkan but has a sounds in UCES01290 in non softgpu.
@Feregorn selecting software with frameskip should help  performance 

Framedump seems don't work in this game
[UCES01290-OpenGL.zip](https://github.com/hrydgard/ppsspp/files/11584742/UCES01290-OpenGL.zip)
[UCES01290-softgpu.zip](https://github.com/hrydgard/ppsspp/files/11584743/UCES01290-softgpu.zip)

**Comment by sum2012** [comment]
This is save status for UCES01290 
[PPSSPP_STATE.zip](https://github.com/hrydgard/ppsspp/files/11584774/PPSSPP_STATE.zip)

**Comment by sum2012** [comment]
Update log to v1.16.6-222-g1e6142d99 with and without softgpu
[ppsspp_opengl_log.zip](https://github.com/hrydgard/ppsspp/files/12905366/ppsspp_opengl_log.zip)
[ppsspp_softgpu_log.zip](https://github.com/hrydgard/ppsspp/files/12905367/ppsspp_softgpu_log.zip)

**Comment by raspael69** [comment]
Hi, I have the same problem with several Buzz games for PSP, like Buzz! Master Quiz and Buzz! ¿What Do You Know About Your Country? among others... The same thing happens, if we choose the 'software rendering' option, the video works, however, the performance drops significantly (Galaxy S23 Ultra).

**Comment by domblah** [comment]
This seems to occur with all Buzz! games, including Quiz World: no video clips are displayed. Intro videos work, but the video clips in questions play without picture, only audio. My current solution is using software rendering. It's maybe a special kind of texture rending of videos where every frame is copied to texture, but idk. Summary these titles has issues with fonts (text) and videos which works on swrender. The text issues (#8594) can be workarounded by skipping gpu readbacks.

<details>
  <summary>Settings (click to expand)</summary>

```
Backend: Vulkan
Rendering resolution: 2x PSP (same on 1x)
Software rendering: no (but yes works)
Antialiasing: off
VSync: on
Skip buffer effects: off
Disable culling: off
Skip GPU Readbacks: No (Default) (Skip works better on text issues, but still no videos)
Lens flare occlusion: Auto (Default)
Lazy texture caching (speedup): off
Spline/Bezier curves quality: High (Default)
Lower resolution for effects: off
Render duplicate frames to 60 Hz: on
Buffer graphics command: Up to 2
Hardware transform: on
Software skinning: on
Hardware tessellation: off
GPU texture upscaler (fast): Tex2xBRZ (same with off and no upscaler)
Anisotrophic filtering: 16x
Texture filterin: auto
Fast memory (unstable): on
Ignore bad memory accesses: on
I/O timing method: Simulate UMD delays
Force real clock sync (slower, less lag): off
Change emulated PSP's CPU clock (unstable): auto
```
</details>

v1.20.4, macOS, M4
